app/docker/views/containers/create/createcontainer.html

```diff
@@ -503,11 +503,10 @@
               <!-- !privileged-mode -->
               <!-- runtimes -->
               <div class="form-group">
-                <label for="container_runtime" class="col-sm-2 col-lg-1 control-label text-left">Runtime</label>
-                <div class="col-sm-1">
-                  <select class="form-control" ng-model="config.HostConfig.Runtime" id="container_runtime">
-                    <option selected disabled hidden value="">Select a runtime</option>
```

Comment:
What about simply adding another option in here with a value of `''` ? And then set the `$scope.config.HostConfig.Runtime = ''` in the controller?

Reply:
Definitely better and simpler solution. And it's working aswell.
Thing is, when value is empty in .html, it's considered as `null` in the ng-model. But that was the default runtime value until 1.19.2, no it's working aswell.
Default value won't be an empty string tho.